sirepo/package_data/static/json/schema-common.json

```diff
@@ -126,7 +126,9 @@
         "deleteFile": "/delete-file",
         "deleteSimulation": "/delete-simulation",
         "downloadDataFile": "/download-data-file/<simulation_type>/<simulation_id>/<model>/<frame>/?<suffix>",
-        "downloadFile": "/download-file/<simulation_type>/<simulation_id>/<filename>",
+        "downloadLibFile": "/download-lib-file/<simulation_type>/<filename>",
+        "downloadRunFile": "/download-run-file/<simulation_type>/<simulation_id>/<model>/<frame>/?<suffix>",
+        "downloadFile": "/download-file/<simulation_type>/<filename>",
```

Comment:
For backwards compatibility /download-file/ should have simulation_id.

Reply:
[ab97c69](https://github.com/radiasoft/sirepo/pull/6348/commits/ab97c6945b6e222591d8b4cab2721e5308a96982), [f1bb3a2](https://github.com/radiasoft/sirepo/pull/6348/commits/f1bb3a233050a89f9eb071756adf7d55ee663fe1)